tck/src/main/java/org/eclipse/microprofile/openapi/tck/AppTestBase.java

```diff
@@ -48,20 +50,24 @@
     private static String username;
     private static String password;
     private static Map<String, String> headers = new HashMap<>();
-
+        
     @Deployment(name = "proxy")
     public static WebArchive createProxy() {
+        
+        MavenResolverSystem resolver = Maven.resolver();  
+        resolver.loadPomFromFile("pom.xml");  
+        File[] files = resolver.resolve("org.apache.httpcomponents:httpcore:4.4.4",
+                "com.fasterxml.jackson.core:jackson-core:2.8.6",
```

Comment:
In the `tck/pom` we have variables such as {{httpclient.version} and {jackson.version}, but here we're hardcoding them.  Can we investigate a way to use the variables here too?  That way we only need to change a single place when updating versions in the future.